frontend/src/app/processes/processes.component.ts

```diff
@@ -78,4 +78,11 @@ export class ProcessesComponent implements OnInit, OnDestroy {
   public resetPage(){
     window.location.reload();
   }
+
+  public reset(){
+    this.selectedApproach = 'None';
+    this.selectedConsent = 'None';
+    this.selectedDepth = 0.0;
+    this.loadGraph();
```

Comment:
Wouldn't it be better to just let the user do a manual reload after possibly selecting some filters after the reset?
From the UX perspective I'd probably be annoyed if the graph was automatically reloaded as this will also change the data that can be selected from the consents and banks fields.